### PR TITLE
add user model

### PR DESCRIPTION
**Pull request by williambaibai**
- added a preliminary user model compatible with Google login
- added some missing code for the skeleton for the database
image

### CODE REVIEW

File: src/app/__init__.py [hunk 1]
```diff
@@ -13,3 +13,14 @@
 
 # Import + Register Blueprints
 blueprint = Blueprint('controllers', __name__, url_prefix='/api/v1')
+
+from app.controllers._all import controllers as controllers
```
Comment (cswenberg) [inline src/app/__init__.py]: if you are keeping the name the same, you don't need `as controllers`

File: src/app/controllers/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from flask import request, render_template, jsonify, redirect
```
Comment (cswenberg) [inline src/app/controllers/__init__.py]: alphabetize

File: src/run.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from app import app
```
Comment (meganle) [inline src/run.py]: Can we somehow incorporate this into `app.py` which runs the GraphQL endpoint at `/`? That way we just have one file that runs the server

**Comment by williambaibai** [comment]
@meganle @cswenberg @orkosinha @johnathananderson addressed PR issues, ready for review

**Comment by young-k** [comment]
@williambaibai Make sure your commits are in sentence case, so `Add user model`, also make sure to delete branches after merging!

**Pull request merged**